scipy/stats/tests/test_distributions.py

```diff
@@ -5219,8 +5241,7 @@ def test_cdf_against_r(self, r_case_result):
 
     @pytest.mark.slow
     def test_moment_vectorization(self):
-        # Test moment broadcasting. Calls `_munp` directly because
-        # `rv_continuous.moment` is broken at time of writing. See gh-12192
+        # Test moment broadcasting. Calls `_munp` directly         # `rv_continuous.moment` is broken at time of writing. See gh-12192
```

Comment:
Oops I don't think this was intended.

Reply:
Correct. Didn't notice. 
```suggestion
        # Test moment broadcasting. Calls `_munp` directly because
        # `rv_continuous.moment` is broken at time of writing. See gh-12192
```